RELEASE_NOTES.md

```diff
@@ -12,6 +12,14 @@
 
 * The `LogicalMeter` no longer takes a `component_graph` parameter.
 
+* `BatteryPool` metric streaming interfaces have changed for `soc`, `capacity` and `power_bounds`:
+
+  ``` python
```

Comment:
syntax should be ``` python 

And this makes me realize that we aren't testing the RELEASE_NOTES examples yet

Reply:
@leandro-lucarella-frequenz do we want to test that? If so this example would fail of course

Reply:
Well, that could be solved by either commenting the wold syntax/way or by having an extra ``` wrapped block that is not marked as python

Reply:
removed the extra space.

Reply:
I think that adds more noise than it helps. Examples in the RELEASE NOTES are a one-off thing, if you checked the time you write them, then they don't need to be kept up to date ever again. I really think that would make writing release notes more annoying and harder to read with very little gain.